js/ui/appActivation.js

```diff
@@ -27,6 +27,9 @@ const SPLASH_SCREEN_TIMEOUT = 700; // ms
 const DEFAULT_MAXIMIZED_WINDOW_SIZE = 0.75;
 const LAUNCH_MAXIMIZED_DESKTOP_KEY = 'X-Endless-LaunchMaximized';
 
+const LAUNCH_REASON_UNINTERESTING = 'uninteresting';
+const LAUNCH_REASON_CODING_BUILDER = 'coding-builder';
```

Comment:
Sure - I was thinking of something similar  but must have forgotten to get around to this.